sktime/classification_sklearn/__init__.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
```

Comment:
not sure if `classification_sklearn` is the best name, but its not a blocker for me

Reply:
well, any better ideas for the name of the folder or location?
I just think it should be in some dedicate location where sklearn classifiers are

Reply:
I'm fine with either location as long as there is a clear naming convention and rationale.

Reply:
Package has changed to classification_sklearn to classification.sklearn